examples/included-as-non-root-mapped/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM cypress/included:3.8.0
+
+# "root"
+RUN whoami
+# uid=0(root) gid=0(root) groups=0(root)
+# meaning root
+RUN id
+
+# there is a built-in user "node" that comes from the very base Docker Node image
+# we are going to recreate this user and give it _same id_ as external user
+# that is going to run this container.
+ARG USER_ID=501
+ARG GROUP_ID=999
+
+# if you want to see all existing groups uncomment the next command
+# RUN cat /etc/group
+
+RUN groupadd -g ${GROUP_ID} appuser
+RUN useradd -r -u ${USER_ID} -g appuser appuser
```

Comment:
This needs the `-l` or `--no-log-init` option to avoid filling up the disk due to  moby/moby#5419 when a large uid is used.